src/Microsoft.AspNet.Mvc.Core/ResponseCacheLocation.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// Determines the value for the "Cache-control" header in the response.
+    /// </summary>
+    public enum ResponseCacheLocation
+    {
+        /// <summary>
+        /// Cached in both proxies and client
+        /// </summary>
+        Any = 0,
```

Comment:
Can we use the remarks here to document exactly what header values these map to?